integrations/appboy/lib/index.js

```diff
@@ -565,3 +570,14 @@ function isValidProperty(name, value) {
 
   return false;
 }
+
+/**
+ * Validate date:
+ *
+ * @param {*} value Value of the property.
+ *
+ * @return {boolean} <code>true</code> if the value are valid, <code>false</code> otherwise.
+ */
+function isDate(value) {
+  return typeof value === 'object' && value instanceof Date;
```

Comment:
Do we need to check if `typeof value === object`? I'm expecting for everything returning true on `value instanceof Date` to be always an object. (I might be wrong, Javascript is weird). :rocket:

Reply:
Seems like 1st check is not needed, as it's a existing code i kept it due to unknown impact.